lib/_stream_writable.js

```diff
@@ -428,13 +428,11 @@ function onwriteError(stream, state, sync, er, cb) {
     // This can emit finish, and it will always happen
     // after error
     process.nextTick(finishMaybe, stream, state);
-    stream._writableState.errorEmitted = true;
```

Comment:
As said in one of other PRs, these should be kept.

Reply:
This change should not be here at all. Error from my side.